exporter/opencensusexporter/config.go

```diff
@@ -17,13 +17,15 @@ package opencensusexporter
 import (
 	"go.opentelemetry.io/collector/config/configgrpc"
 	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
 )
 
 // Config defines configuration for OpenCensus exporter.
 type Config struct {
 	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
-
 	configgrpc.GRPCClientSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+	exporterhelper.QueueSettings  `mapstructure:"sending_queue"`
+	exporterhelper.RetrySettings  `mapstructure:"retry_on_failure"`
```

Comment:
what about `exporterhelper.TimeoutSettings`?

Reply:
That does not have any effect in this case, because this uses bi-directional streams and I believe (I may be wrong) that timeout does not have any effect when sending a message (this is not an RPC, just a message in a stream).

Reply:
ok, I'm asking just because I have seen that you've added it along with `QueueSettings` and `RetrySettings` in another exporter